examples/Example17/include/PrimaryGeneratorAction.hh

```diff
@@ -54,12 +55,18 @@ public:
   void SetDefaultKinematic();
   void SetRndmBeam(G4double val) { fRndmBeam = val; }
```

Comment:
You may need to set a seed for the random number generator, and a function to initialize the generator from this seed. We need to be able to produce reproducible but different sequences, and to replay the same sequence for G3ant4/AdePT